subprojects/packagefiles/xz-5.2.5/meson.build

```diff
@@ -154,6 +154,23 @@ cdata.set('HAVE_MF_BT4', 1)
 cdata.set('HAVE_MF_HC3', 1)
 cdata.set('HAVE_MF_HC4', 1)
 
+has_sysconf_cpu = cc.compiles('''#include <unistd.h>
```

Comment:
Could you please extract this change and send a separate PR?

Reply:
https://github.com/rizinorg/rizin/pull/2801